assets/js/toggle_view.js

```diff
@@ -0,0 +1,74 @@
+
+
+// Elements for Listeners
+jwt_nav = document.getElementById("jwt-nav")
+pkce_nav = document.getElementById("pkce-nav")
+oidc_auth_success_button = document.getElementById("oidc_auth_success_button")
+oidc_auth_error_button = document.getElementById("oidc_auth_error_button")
+
+// Selectors
+oidc_auth_success_button.selector = "oidc_auth"
+oidc_auth_error_button.selector = "oidc_auth"
```

Comment:
If these are custom, I'd suggest using `dataset` (`data-` attributes) instead. These would be flagged by TypeScript if/when we integrate that into the project.

```suggestion
// Selectors
oidc_auth_success_button.dataset.selector = "oidc_auth"
oidc_auth_error_button.dataset.selector = "oidc_auth"
```

Better yet, maybe that can be added in the markup directly as `data-` attributes?